### DISCUSSION TITLE
Which of the rich colors corresponds to the "success" and "error" colors in textual?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#5060] [Q&A]
Which of the rich colors corresponds to the success and incorrect colors in textual?

rich color:
https://rich.readthedocs.io/en/stable/appendix/colors.html#:~:text=Standard%20Colors.%20The%20following%20is%20a%20list%20of

thanks

**Comment by darrenburns** [comment]
Textual has it's own design system, unrelated to the colours in the Rich docs you linked.

Success is `#4EBF71` and error is `#BA3C5B` in Textual. You can use these hex codes in Rich.

**Reply by Yurii-huang** [reply]
> Textual has it's own design system, unrelated to the colours in the Rich docs you linked.
> 
> Success is and error is in Textual. You can use these hex codes in Rich.`#4EBF71``#BA3C5B`

Okay
